config/appsignal.yml

```diff
@@ -18,3 +18,4 @@ default: &defaults
 production:
   <<: *defaults
   active: true
+  revision: "<%= ENV["HATCHBOX_REVISION"] %>"
```

Comment:
https://www.hatchbox.io/announcements/22 - confirmed this is available in production